unittests/test_exec_finder.py

```diff
@@ -39,7 +43,11 @@ def test_find_in_two_places(tmpdir):
     w2.remove()
     assert ef.get_executable_path("abc.aplx") == str(w1)
     w1.remove()
-    assert ef.get_executable_path("abc.aplx") is None
+    try:
+        ef.get_executable_path("abc.aplx")
+        return False
+    except Exception:
+        return True
```

Comment:
This is _not_ how you assert that an exception happened!

Reply:
fixed